vsutil/__init__.py

```diff
@@ -251,6 +251,65 @@ def is_image(filename: str, /) -> bool:
     return types_map.get(path.splitext(filename)[-1], '').startswith('image/')
 
 
+def scale(value: float, 
+          input_depth: int = 8, 
+          output_depth: int = 32, 
+          range_in: Union[int, Range] = False, 
```

Comment:
Why use booleans here and not ints? I think this is needlessly confusing.
I complain because `True` and `False` are ints, yes but they have additional connotations.